src/trapfunc.cpp

```diff
@@ -1231,7 +1249,18 @@ bool trapfunc::ledge( const tripoint &p, Creature *c, item * )
                                     _( "<npcname> flaps their wings and flutters down gracefully." ) );
     } else if( you->has_active_bionic( bio_shock_absorber ) ) {
         you->add_msg_if_player( m_info,
-                                _( "You hit the ground hard, but your grav chute handles the impact admirably!" ) );
+                                _( "You use your " ) );
```

Comment:
This message makes no sense. What are you using here?

Reply:
Thanks, probably a mistake from when I squashed the original commits